### ISSUE TITLE
Rework when Mage Stats are displayed

### ISSUE THREAD

**Issue by sIKE23** [sIKE23/Mage-Wars#351]
Mage Stats should be revealed when the mage card is revealed not when the deck is being validated. The player (in a two player game) loading their deck second can make an educated guess as to what mage they will be facing by letting the other player load their Mage first.

**Comment by sIKE23** [comment]
@ACG8 

Right now this is what is fed into the chat log on Mage Reveal
```
sIKE23 prepares a Spell from their Spellbook by placing a card face down on the table.
Anvil Throne Warlord spends Action Marker
Anvil Throne Warlord readies Action Marker
Anvil Throne Warlord readies Quickcast Marker
sIKE23 Channeling is set to 9 and Mana is set to 19 and Life set to 34
```
I would like to pretty this up a bit.....any thoughts / recommendations?

**Comment by ACG8** [comment]
You mean like flavor text? We could add a check to card reveal that checks whether it is a mage, then retrieves a custom message (possibly one of several per mage) based on which mage it is. For instance:

```A warlord of the Anvil Throne enters the arena, a look of determination on his face. "Yer ready then?" he grunts.```

```A warlock of Adramelech appears in a burst of flame. She is laughing maniacally.```

```A druid enters the arena, flowers blossoming at her feet.```

**Comment by sIKE23** [comment]
Yeah that would be very cool! You are so good at stuff like that! I also
need to figure out if I should suppress the maker comments.

One other thought should we store that "the Mage Reveal event" has
happened? The code in question is called from flipcard() and I think we
want something like:

*if MageRevealEvent == "False" mageSetup()*

That way it is only triggered once, the old way was done at ondeckloaded

//FC

**Comment by sIKE23** [comment]
Your proposal, is it a lot of work?

**Comment by ACG8** [comment]
No; I'll handle it, I just have a lot of things on my plate this week. But adding the check should not be very difficult.
